go-controller/pkg/ovn/base_secondary_layer2_network_controller.go

```diff
@@ -0,0 +1,254 @@
+package ovn
```

Comment:
only purpose of this is to move the retry framework into a more generic  base package so you can call it for different layer 2 secondary network types?

Reply:
this was changed this way based on @jcaamano's comments https://github.com/ovn-org/ovn-kubernetes/pull/3380#discussion_r1103130738. The layer2/localnet network controller do have a lot of code in common.